### DISCUSSION TITLE
Chinese disorder code

### DISCUSSION THREAD

**Discussion by xduanx** [PyMySQL/PyMySQL#964] [Q&A]
**Describe the bug**
Chinese disorder code

**To Reproduce**
(venv) [cloud@localhost sql_test]$ LANG=C
(venv) [cloud@localhost sql_test]$ mysql -uduan -pAa123456 -e 'INSERT INTO db1.t1 VALUES("文件存储AAAA11111111");'
(venv) [cloud@localhost sql_test]$ python test_sql.py 
('\xc3\xa6\xe2\x80\x93\xe2\x80\xa1\xc3\xa4\xc2\xbb\xc2\xb6\xc3\xa5\xc2\xad\xcb\x9c\xc3\xa5\xe2\x80\x9a\xc2\xa8AAAA11111111',)
æ–‡ä»¶å­˜å‚¨AAAA11111111

Schema:

```
CREATE DATABASE ...
CREATE TABLE ...
```

Code:
(venv) [cloud@localhost sql_test]$ cat test_sql.py
import pymysql
import os

def test1():
    conn = pymysql.connect(host='192.168.226.217', user='duan', password='Aa123456', database='db1', port=3306)
    cur = conn.cursor()
    sql = 'select * from t1'
    count = cur.execute(sql)
    ret = cur.fetchall()
    for i in ret:
        print i
        print i[0]
    cur.close()
    conn.close()

if __name__ == '__main__':
    test1()

**Expected behavior**

(venv) [cloud@localhost sql_test]$ mysql -uduan -pAa123456 -e 'SELECT * FROM db1.t1;'
+--------------------------+
| name                     |
+--------------------------+
| 文件存储AAAA11111111 |
+--------------------------+

**Environment**
 - OS: CentOS Linux release 7.8.2003 (Core)
 - Server and version: 5.5.65-MariaDB MariaDB Server
 - PyMySQL version: 0.10.1

**Additional context**
the all os' env LANG is LANG=C, so I have to use LANG=C.

**Comment by canotf** [comment]
中文乱码改gbk，亲测可以
